### PR TITLE
Add print of transaction results

### PR DESCRIPTION
**Pull request by roytman**
Signed-off-by: Alexey Roytman <roytman@il.ibm.com>

### CODE REVIEW

**Comment by hunchback** [comment]
please just use json.Marshal as I did with request, it will automatically remove fields which are nil

**Comment by roytman** [comment]
> 
> 
> please just use json.Marshal as I did with request, it will automatically remove fields which are nil

OK, will do it

File: dist/ovn-kubernetes/dist/images/ovnkube.sh [hunk 1]
```diff
@@ -1211,7 +1211,8 @@ ovs-metrics() {
 etcd () {
   echo "================= start etcd server ============================ "
   /usr/local/bin/etcd --data-dir /etc/openvswitch/ --listen-peer-urls http://localhost:2480 \
-  --listen-client-urls http://localhost:2479 --advertise-client-urls http://localhost:2479
+  --listen-client-urls http://localhost:2479 --advertise-client-urls http://localhost:2479 \
+  --max-txn-ops 400
```
Comment (hunchback) [inline dist/ovn-kubernetes/dist/images/ovnkube.sh:1215]: this conflicts with my fix (of splitting large transactions)

File: pkg/libovsdb/notation.go [hunk 2]
```diff
@@ -170,8 +170,8 @@ func NewTransact(params []interface{}) (*Transact, error) {
 
 // TransactResponse represents the response to a Transact Operation
 type TransactResponse struct {
-	Result []OperationResult `json:"result"`
-	Error  string            `json:"error"`
+	Result []OperationResult `json:"result,omitempty"`
+	Error  *string            `json:"error,omitempty"`
```
Comment (hunchback) [inline pkg/libovsdb/notation.go:174]: alignment appears wrong

File: pkg/ovsdb/handler.go [hunk 3]
```diff
@@ -44,7 +44,7 @@ func (ch *Handler) Transact(ctx context.Context, param []interface{}) (interface
 	txn := NewTransaction(ch.etcdClient, req)
 	txn.schemas = ch.db.GetSchemas()
 	txn.Commit()
-	klog.V(5).Infof("Transact response %v", txn.response)
+	klog.V(5).Infof("Transact response %s", &txn.response)
```
Comment (hunchback) [inline pkg/ovsdb/handler.go:47]: why pointer? it should work also without ampercent

Comment (roytman) [reply pkg/ovsdb/handler.go:47]: The String method defined for `*TransactResponse`. Without ampercent it doesn't work, or requires explicitly call String()

File: pkg/libovsdb/notation.go (hunk 2 above)
Comment (roytman) [reply pkg/libovsdb/notation.go]: fixed

File: dist/ovn-kubernetes/dist/images/ovnkube.sh (hunk 1 above)
Comment (roytman) [reply dist/ovn-kubernetes/dist/images/ovnkube.sh:1215]: I rebased it with the master.

**Pull request merged**